### PR TITLE
Documentation for Hid injection Driver

### PR DESCRIPTION
**Pull request by ooeygui**

### CODE REVIEW

File: en-US/win10/samples/HidInjection.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  1. You can now build the solution.
+
+### Build the Update package
+  1. Open the ```Deployment and Imaging Tools Environment``` from the Start Menu with administrative privledges (Search for the program, then right click on it and select ```Run As Administrator```)
+  1. In the command window, cd into your project directory.
+  1. cd into the driver directory.
+  1. run the script ```CreateDriverPackage.cmd```
+  
+### Copying the HID Injector to your device
+  1. Open a network share on your IoT Core device by opening the Run dialog (Win-R), then entering \\```IP for your IoT Core device\c$```. Enter credentials if prompted.
```
Comment (alecont) [inline en-US/win10/samples/HidInjection.md]: I think markdown gets confused with the double slash (at least, I see only one slash in the rich preview here in GitHub).

File: en-US/win10/samples/HidInjection.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Building and Deploying the HID Injector
+  1. Be sure you've installed the Visual Studio update and the Windows 10 Driver Development kit before continuing.
+  1. On your development machine, use Windows Explorer to navigate to the folder where you downloaded or cloned samples.
+  1. Open the project located ```Samples\HidInjector\HidInjector.sln```.
+  1. Select the architecture you intend to deploy to: ARM for Raspberry Pi or Dragon Board, x86 for MinnowBoard Max.
+  1. You can now build the solution.
+
+### Build the Update package
+  1. Open the ```Deployment and Imaging Tools Environment``` from the Start Menu with administrative privledges (Search for the program, then right click on it and select ```Run As Administrator```)
```
Comment (alecont) [inline en-US/win10/samples/HidInjection.md]: typo: privledges -> privileges

**Comment by alecont** [comment]
minor comments, but great doc page! :-)

File: en-US/win10/samples/HidInjection.md (hunk 1 above)
Comment (ivorb) [inline en-US/win10/samples/HidInjection.md]: Can you just include the double slash in the inline code?

**Comment by alecont** [comment]
looks good

**Pull request merged**